homeassistant/components/jewish_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
```

Comment:
we can remove the 
![image](https://github.com/user-attachments/assets/952cd058-9b20-4de2-b968-eb5cbd46c87a)
parenthesis here

Reply:
Done. See home-assistant/home-assistant.io#39998